actool.mk

```diff
@@ -0,0 +1,16 @@
+$(call setup-stamp-file,ACTOOL_STAMP)
+
+# variables for makelib/build_go.mk
+BGB_PKG_IN_REPO := Godeps/_workspace/src/github.com/appc/spec/actool
```

Comment:
Yeah, the comment above is a bit outdated (should mention `build_go_bin.mk` instead). It is basically a namespaced variable, so it does not accidentally override something else.